crates/sc-consensus-subspace/src/verifier.rs

```diff
@@ -569,6 +558,19 @@ where
         } = checked_header;
 
         let slot = pre_digest.slot();
+        // Estimate what the "current" slot is according to sync target since we don't have other way to know it
+        let diff_in_blocks = self
+            .sync_target_block_number
+            .load(Ordering::Relaxed)
+            .saturating_sub(BlockNumber::from(*pre_header.number()));
+        let slot_now = if diff_in_blocks > 0 {
```

Comment:
So this would essentially, be zero when we are chain is fully synced and we want to check the equivocation for the latest head then ?

Reply:
Yes, Substrate checks that `slot_now - slot` should be a relatively small number (constant is set to 1000 slots right now), if exceeded, equivocation check is skipped.